SparkPost/matchbox
# FE-826 Update Popover Component

### What Changed
- Restyles `Popover`
- Adds new `id` prop for `Popover`
- Adds new `as` prop for `Popover`, functions the same way as `wrapper`
- Removes the `portalId` prop from `Popover`
- Removes the `fixed` prop from `Popover`, since popover content no longer uses portals
- Adds new exported hook `useWindowEvent`

![Screen Shot 2020-04-01 at 7 58 40 PM](https://user-images.githubusercontent.com/3903325/78197418-46f83280-7453-11ea-9215-b0b2d1a599ad.png)

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify popover stories
- Take a look at cypress tests and verify all window events are tested
- Verify other components or stories that use Popover – Page and Actionlist


### PR Checklist
